workspaces/adventure-pack/goodies/typescript/Iterator.prototype.take/index.test.ts

```diff
@@ -0,0 +1,89 @@
+import { describe, expect, it } from "@jest/globals";
```

Comment:
Btw, feel free to update the contributors list in the Adventure Pack's `package.json` to include your name! (i.e. the file `workspaces/adventure-pack/package.json`)